### PR TITLE
Task/more random cleanup

### PR DESCRIPTION
**Pull request by uliluckas**
**Best reviewed commit by commit**

## Changes

- Lint cleanup - Hopefully only indisputable changes.
- Fix sick notifications on currently disabled daily processing branch. @falkorichter please review this.

### CODE REVIEW

**Comment by falkorichter** [comment]
I´m confused should 1d8fc61595ab05ecff6d9f9b33d8dcf5e9348a3c not be in this PR? I don´t see it in the list of commit.

**Review by falkorichter — CHANGES_REQUESTED** [review]
check the comments in https://github.com/austrianredcross/stopp-corona-android/commit/1d8fc61595ab05ecff6d9f9b33d8dcf5e9348a3c#diff-b81c6cf0dc0abfc66854d4f595216581R208

File: app/src/main/java/at/roteskreuz/stopcorona/model/repositories/DiagnosisKeysRepository.kt [hunk 1]
```diff
@@ -187,9 +187,17 @@ class DiagnosisKeysRepositoryImpl(
         firstRedDay?.let { _ ->
             firstYellowDay?.let { _ ->
                 quarantineRepository.receivedWarning(WarningType.YELLOW, timeOfContact = firstYellowDay)
-            } ?: quarantineRepository.revokeLastYellowContactDate()
+            }
 
             quarantineRepository.receivedWarning(WarningType.RED, timeOfContact = firstRedDay)
+
+            // Only revoce quarantines after all new quarantines are known.
```
Comment (MidnightDesign) [inline app/src/main/java/at/roteskreuz/stopcorona/model/repositories/DiagnosisKeysRepository.kt]:
```suggestion
            // Only revoke quarantines after all new quarantines are known.
```

**Review by falkorichter — APPROVED** [review]
off you go!

**Pull request merged**